client/my-sites/plans/jetpack-plans/product-store/view-filter/style.scss

```
@@ -0,0 +1,57 @@
+.jetpack-product-store .jetpack-product-store__view-filter .jetpack-product-store__view-filter--toggle.segmented-control {
```

Can we simplify the selectors in this file?

```suggestion
.jetpack-product-store__view-filter--toggle.segmented-control {
```